### PR TITLE
PMG-145: disable messages for deceased members

### PR DESCRIPTION
**Pull request by waracci**
**Before:**

Screenshot from 2022-01-20 10-46-25

**After:**

Screenshot from 2022-01-20 10-47-53

### CODE REVIEW

**Comment by waracci** [comment]
I also included the styling for former parties

**Before:**

Screenshot from 2022-01-24 08-38-13


**After:**

Screenshot from 2022-01-24 08-39-51

File: pombola/south_africa/templates/core/person_detail.html [hunk 1]
```diff
@@ -159,8 +159,22 @@ <h3 class="mp-block__title">Political party:</h3>
...
-                  <a href="{{ party.get_absolute_url }}"
-                    class="party-membership party-membership--{{ party.kind.name | slugify }}">{{ party.name }}</a></li>
+                  <a href="{{ party.get_absolute_url }}" {% for identifier in party.identifiers.all %}
+                    data-identifier-{{ identifier.scheme }}="{{ identifier.identifier }}" {%endfor%}
+                    class="text-link w-inline-block">
+                    <p class="text-link__text">
+                      {{ party.name }}
+                    </p>
+                    <div class="text-link__icon">
+                      <div class="svg-icon w-embed"><svg xmlns="http://www.w3.org/2000/svg" aria-hidden="true"
```
Comment (paulmwatson) [inline pombola/south_africa/templates/core/person_detail.html]: Let's use an external SVG file please

File: pombola/writeinpublic/templates/writeinpublic/messages.html [hunk 2]
```diff
@@ -38,21 +38,23 @@
...
-            <path fill="currentColor"
-              d="M497.9 142.1l-46.1 46.1c-4.7 4.7-12.3 4.7-17 0l-111-111c-4.7-4.7-4.7-12.3 0-17l46.1-46.1c18.7-18.7 49.1-18.7 67.9 0l60.1 60.1c18.8 18.7 18.8 49.1 0 67.9zM284.2 99.8L21.6 362.4.4 483.9c-2.9 16.4 11.4 30.6 27.8 27.8l121.5-21.3 262.6-262.6c4.7-4.7 4.7-12.3 0-17l-111-111c-4.8-4.7-12.4-4.7-17.1 0zM124.1 339.9c-5.5-5.5-5.5-14.3 0-19.8l154-154c5.5-5.5 14.3-5.5 19.8 0s5.5 14.3 0 19.8l-154 154c-5.5 5.5-14.3 5.5-19.8 0zM88 424h48v36.3l-64.5 11.3-31.1-31.1L51.7 376H88v48z">
-            </path>
-          </svg></div>
-      </div>
-      <p>
-        Write a public message to this politician
-      </p>
-    </a>
+    {% if person.is_current_member_of_national_assembly %}
```
Comment (paulmwatson) [inline pombola/writeinpublic/templates/writeinpublic/messages.html]: There is a `date_of_death` field used in other parts of the site which sounds more appropriate for deceased members than using whether they are a current member of National Assembly?

File: pombola/south_africa/templates/core/person_detail.html (hunk 1 above)
Comment (waracci) [reply pombola/south_africa/templates/core/person_detail.html]: done

File: pombola/writeinpublic/templates/writeinpublic/messages.html (hunk 2 above)
Comment (waracci) [reply pombola/writeinpublic/templates/writeinpublic/messages.html]: good call, changed it

**Pull request merged**